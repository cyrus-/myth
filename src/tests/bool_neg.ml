let s = Bool_decls.s ^ "

let bool_neg : bool -> bool |>
  { True => False
  ; False => True } = ?"
