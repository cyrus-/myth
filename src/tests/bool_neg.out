(* Generated Thu Nov 13 00:35:22 2014 *)
let bool_neg : bool -> bool =
  fun (b1:bool) -> match b1 with
                     | True -> False
                     | False -> True
;;
